.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
Maybe add a default label. Or is this not only for bugs?

Reply:
Default label will be bug. 
AFAIK people can still create issues freely e.g. feature requests,...